common/static/common/js/spec/discussion/view/discussion_thread_list_view_spec.js

```diff
@@ -261,14 +261,23 @@
             });
         });
         describe("search alerts", function() {
-            var testAlertMessages;
+            var testAlertMessages, getAlertMessagesAndClasses;
 
             testAlertMessages = function(expectedMessages) {
                 return expect($(".search-alert .message").map(function() {
                     return $(this).html();
                 }).get()).toEqual(expectedMessages);
             };
 
+            getAlertMessagesAndClasses = function () {
+                return $(".search-alert").map(function() {
```

Comment:
inconsistent spacing between `function` and `()` on these two lines

(As mentioned above, new styleguide/inter will catch stuff like this, the preferred style is no space (`function() {`)  - but just be consistent one way or another and the autofixer will get stuff like this when we turn it on in this repo)